src/main/java/io/github/robwin/swagger/test/ConsumerDrivenValidator.java

```diff
@@ -233,8 +233,6 @@ private void validateOperation(Operation actualOperation, Operation expectedOper
                 //Validate responses
                 validateResponses(actualOperation.getResponses(), expectedOperation.getResponses(), httpMethod, path);
             }
-        }else{
-            softAssertions.assertThat(actualOperation).as(message).isNull();
```

Comment:
deleted, because it seems to be copied from DocumentationDrivenValidator where it makes sense